app/components/stepper.js

```diff
@@ -21,17 +25,28 @@ export default class StepperComponent extends Component {
   setIsValid = (newVal) => (this.isValid = newVal);
   setIsPreValid = (newVal) => (this.preValid = newVal);
 
+  constructor() {
+    super(...arguments);
+    window.onpopstate = () => {
+      this.currentStep = Number(
+        +new URLSearchParams(window.location.search).get('step')
```

Comment:
If it is possible to do the router service, to get a query param use that.
```js
    window.onpopstate = () => {
      const queryParams = this.router.currentRoute.queryParams;
      this.currentStep = Number(queryParams.step);
    };
```

Reply:
not working, its always return 0

Reply:
You can use `this.args.step` here instead of using `window`, correct if I am wrong.